keras/layers/containers.py

```diff
@@ -20,6 +20,8 @@ def __init__(self, layers=[]):
         self.params = []
         self.regularizers = []
         self.constraints = []
+        self.cost_updates = []
```

Comment:
The general term used throughout Keras is "loss" (as well as "objective" for the objective functions); I think we should keep it that way and replace "cost" with "loss" in the variables.